src/history.js

```diff
@@ -297,6 +297,13 @@ class History extends Emitter {
 
     this.size = size
   }
+
+  toJSON() {
+    return {
+      size: this.size,
+      root: this.root
```

Comment:
if the action tree is multi-branched, should there be something here that denotes the head?